docs/content/en/integration/openid-connect/budibase/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Budibase"
+description: "Integrating Budibase with the Authelia OpenID Connect 1.0 Provider."
+lead: ""
+date: 2023-11-12T15:50:35+00:00
+draft: false
+images: []
+menu:
+  integration:
+    parent: "openid-connect"
+weight: 620
+toc: true
+community: true
+---
+
+## Tested Versions
+
+- [Authelia]
+  - [v4.37.5](https://github.com/authelia/authelia/releases/tag/v4.37.5)
```

Comment:
```suggestion
  - [v4.38.0](https://github.com/authelia/authelia/releases/tag/v4.38.0)
```

We're only accepting contributed docs for the upcoming 4.38.0 which will be the basis for this change and some changes to the config example.